### PR TITLE
Enhance source code

### PR DESCRIPTION
**Pull request by donho**

### CODE REVIEW

File: PowerEditor/WinControls - Shortcut.lnk
Comment (alankilborn) [inline PowerEditor/WinControls - Shortcut.lnk:1]: A shortcut file...in the source code??

**Comment by ozone10** [comment]
@donho 
maybe include this small modification in this PR:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/8d3d8aa6429a7408ca47c2fc15e0b8729992ef69/PowerEditor/src/WinControls/Grid/BabyGrid.cpp#L677

could be deleted since it's not used anyway.

File: PowerEditor/src/WinControls/DockingWnd/DockingCont.cpp [hunk 1]
```diff
@@ -1094,8 +1076,8 @@ void DockingCont::drawTabItem(DRAWITEMSTRUCT *pDrawItemStruct)
 		if ((hImageList != NULL) && (iPosImage >= 0))
 		{
 			// Get height of image so we
-			IMAGEINFO	info		= {};
-			RECT &		imageRect	= info.rcImage;
+			IMAGEINFO info = {};
```
Comment (ozone10) [inline PowerEditor/src/WinControls/DockingWnd/DockingCont.cpp:1079]: for consistency I would use `IMAGEINFO info{};`, same for `TCITEM` variables below

**Comment by casey-chesshir** [comment]
your build is failing because this says cons instead of const 
image

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/e6558a36188532bbf12f5aee985a96b27c9a23ad

**Pull request closed without merging**